Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-5554.dfy

```diff
@@ -0,0 +1,13 @@
+// RUN:  %dafny -compile:4 -compileTarget:cs "%s"
+
+datatype MultisetContainer = EmptySet | BooleanMultiset(containerSet: multiset<bool>)
+
+method Main() {
+    var initialMultiset := multiset{false};
+    
+    for iteration := 0 to 98 {
+        var multisetInstance := BooleanMultiset(initialMultiset); 
+        initialMultiset := initialMultiset + multisetInstance.containerSet;  
+    }
+    print initialMultiset > initialMultiset;
+}
```

Comment:
needs new line at EOF

Reply:
Btw, do you know why 98, or perhaps a better question is: is there a limit or does BigInteger support arbitrary sized integers?

Reply:
BigIntegers are unbounded, see https://learn.microsoft.com/en-us/dotnet/api/system.numerics.biginteger?view=net-8.0
In this example, the number of elements in the multiset after i iterations of the loop is 2^i. So there is an overflow in the computation of the size of the multiset (within the translation of the '>' operator in CS) as soon as the loop runs 31 times. Hence I change the 98 bound to 31.